ast/src/parsed/mod.rs

```diff
@@ -1150,6 +1212,28 @@ impl<E> Children<E> for IfExpression<E> {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct StructExpression<E = Expression<NamespacedPolynomialReference>> {
+    pub name: String,
```

Comment:
Just thinking aloud here: Shouldn't this be a SymbolPath? I know this will might create problems later on where we only work with direct Strings...